src/lib/validations/is-multi-site.js

```diff
@@ -15,27 +15,37 @@ import API from 'lib/api';
 import { trackEventWithEnv } from 'lib/tracker';
 import * as exit from 'lib/cli/exit';
 
+let isMultiSite;
```

Comment:
This will be invalid if `isMultiSiteInSiteMeta` is called multiple times with different input.  If we're going to cache the results, we should do it in a map of the input.